### DISCUSSION TITLE
Context switch plugin

### DISCUSSION THREAD

**Discussion by igorepst** [sayanarijit/xplr#404] [Show and tell]
I wrote a [context switch plugin](https://github.com/igorepst/context-switch.xplr) for `xplr`.
Saves and loads 10 contexts, incl. focused node and selection.
Presents to the user a vertically split layout with a file system on the top and a panel with contexts list. Not initialized contexts have '?' as their path. Current context is marked by 'v'.
Please note, this is a context switch and not a bookmarking plugin, hence every time you open this mode, the stored information for the current context will be changed (current working directory, focused node and selection).
Upon quitting `xplr` the contexts will be cleared.

**Comment by sayanarijit** [comment]
Nice... It might be useful to add a demo video or gif in the readme. What I do is, record gif using `peek` and upload to gifyu.com.

**Comment by sayanarijit** [comment]
Inspired by this plugin, I also implemented [dual pane](https://github.com/sayanarijit/dual-pane.xplr) support.

**Reply by igorepst** [reply]
These are great news, @sayanarijit.
